RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -270,7 +270,8 @@ void PulseShapeFitOOTPileupCorrection::setPUParams(bool   iPedestalConstraint, b
 }
 
 void PulseShapeFitOOTPileupCorrection::setPulseShapeTemplate(const HcalPulseShapes::Shape& ps, bool isHPD) {
-   if( cntsetPulseShape ) return;
+  // comment this otherwise he siPM is not correctly initialized
+  //  if( cntsetPulseShape ) return;
```

Comment:
In practice the pulse template was set only on the first rechit as they were all the same.
Now is going to be initialized for every rec-hit and properly if it belong to HPD/QIE8 or the siPM/QIE10.
So in practice this line will not have any single difference on the output, I haven't checked the CPU time on the 2016.